### DISCUSSION TITLE
Welcome to Alimer Discussions!

### DISCUSSION THREAD

**Discussion by amerkoleci** [amerkoleci/alimer#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MrScautHD** [comment]
Heyy do you has a discord?

**Reply by amerkoleci** [reply]
I do have it, what requests do you have that you cannot ask me here?

**Comment by roy-t** [comment]
I'm Roy T. a tech lead at a large Dutch e-commerce company, much more boring than making video games :). I've started working on game engines when Microsoft launched XNA and I've been blogging and making tutorials a lot. Currently I'm struggling on mixing way to many techniques in my toy render engine and I hope to finally steer it a bit more to making an actual video game though my ideas there are not really concrete :).
